docs/src/use/getting-started.md

```diff
@@ -123,7 +119,7 @@ Before you begin, you must already have a `package.json` file. If you don't, mak
    touch eslint.config.js
    ```
 
-1. Add configuration to the `eslint.config.js` file. Refer to the [Configure ESLint documentation](configure/) to learn how to add rules, custom configurations, plugins, and more.
+2. Add configuration to the `eslint.config.js` file. Refer to the [Configure ESLint documentation](configure/) to learn how to add rules, custom configurations, plugins, and more.
```

Comment:
![Screenshot 2024-09-23 190057](https://github.com/user-attachments/assets/5a1998b0-5303-44d6-b0bc-9b337327160a)

could you fix the counting of the steps here.